Source/_Tests/LibationFileManager.Tests/TemplatesTests.cs

```diff
@@ -406,6 +407,30 @@ public void SeriesFormat_formatters(string template, string expected)
 				.Should().Be(expected);
 		}
 
+		[TestMethod]
```

Comment:
Again: thank you for your unit tests. This kind of dynamic string formatting can't be easily intuited and it's so easy to understand by seeing examples. I'm also glad for the users' sake that we built into the template editor that little sample place where the user can try it out in real time.